package.json

```diff
@@ -2,126 +2,82 @@
-    "babel-eslint": "^10.1.0",
-    "broccoli-asset-rev": "^3.0.0",
-    "broccoli-babel-preset-typescript": "^1.0.1",
-    "ember-auto-import": "^2.2.3",
-    "ember-cli": "~3.28.0",
-    "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-htmlbars": "^6.2.0",
-    "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-sri": "^2.1.1",
-    "ember-cli-terser": "^4.0.2",
-    "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-export-application-global": "^2.0.1",
-    "ember-load-initializers": "^2.1.2",
-    "ember-maybe-import-regenerator": "^1.0.0",
-    "ember-page-title": "^6.2.2",
-    "ember-qunit": "^5.1.5",
-    "ember-resolver": "^8.0.3",
-    "ember-source": "~3.28.6",
-    "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^3.12.0",
-    "ember-try": "^1.4.0",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-ember": "^10.5.7",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-qunit": "^7.3.4",
-    "expect-type": "^0.15.0",
-    "loader.js": "^4.7.0",
-    "npm-run-all": "^4.1.5",
-    "prettier": "^2.6.0",
-    "qunit": "^2.17.2",
-    "qunit-dom": "^2.0.0",
-    "release-it": "^14.11.6",
-    "release-it-lerna-changelog": "^3.1.0",
-    "rimraf": "^3.0.2",
-    "typescript": "^5.0.4",
-    "webpack": "^5"
-  },
-  "engines": {
-    "node": "12.* || 14.* || >= 16"
+    "@arethetypeswrong/cli": "^0.15.0",
+    "@babel/core": "^7.23.6",
+    "@babel/plugin-transform-typescript": "^7.23.6",
+    "@babel/runtime": "^7.17.0",
+    "@embroider/addon-dev": "^4.1.0",
+    "@rollup/plugin-babel": "^6.0.4",
+    "@tsconfig/ember": "^3.0.2",
+    "@types/ember__debug": "^4.0.7",
+    "@typescript-eslint/eslint-plugin": "^6.14.0",
+    "@typescript-eslint/parser": "^6.14.0",
+    "babel-plugin-ember-template-compilation": "^2.2.1",
+    "concurrently": "^8.2.2",
+    "ember-template-lint": "^5.13.0",
+    "eslint": "^8.56.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-ember": "^11.12.0",
+    "eslint-plugin-n": "^16.4.0",
+    "eslint-plugin-prettier": "^5.0.1",
+    "expect-type": "^0.18.0",
+    "prettier": "^3.1.1",
+    "prettier-plugin-ember-template-tag": "^1.1.0",
+    "publint": "^0.2.7",
+    "rollup": "^4.9.1",
+    "rollup-plugin-copy": "^3.5.0",
+    "typescript": "^5.3.3"
   },
   "publishConfig": {
     "registry": "https://registry.npmjs.org"
   },
-  "ember": {
-    "edition": "octane"
-  },
-  "ember-addon": {
-    "configPath": "tests/dummy/config"
-  },
-  "release-it": {
-    "plugins": {
-      "release-it-lerna-changelog": {
-        "infile": "CHANGELOG.md",
-        "launchEditor": true
-      }
-    },
-    "git": {
-      "tagName": "v${version}"
-    },
-    "github": {
-      "release": true,
-      "tokenRef": "GITHUB_AUTH"
-    },
-    "npm": {
-      "publish": false
-    }
-  },
   "volta": {
-    "node": "12.22.3",
-    "yarn": "1.22.11"
+    "node": "20.11.1",
```

Comment:
Node is here v20, but in ci scripts v18... maybe its better to hold sync

Reply:
good point -- node is actually irrelevant now, so I can clean up CI a bit, let it be defaults.